src/include/daos_fs.h

```diff
@@ -998,9 +998,13 @@ dfs_ostat(dfs_t *dfs, dfs_obj_t *obj, struct stat *stbuf);
 #define DFS_SET_ATTR_GID	(1 << 5)
 
 /**
- * set stat attributes for a file and fetch new values.  If the object is a
+ * Set stat attributes for a file and fetch new values.  If the object is a
  * symlink the link itself is modified.  See dfs_stat() for which entries
  * are filled.
+ * Since libdfs does not perform any POSIX permission enforcement, it just
```

Comment:
this is not true technically. DFS does permission enforcement. see:
https://github.com/daos-stack/daos/blob/master/src/client/dfs/obj.c#L19

so we should adjust this slightly to say that since we don't run executables over DFS, we can allow SUID/SGID to be set without being enforced. for the sticky bit, im afraid we need to probably keep that check.